package_manifest.yaml

```diff
@@ -24,13 +24,16 @@ foreman_scl_packages:
     diff_package_skip: false
     diff_package_tags:
       - foreman-nightly-rhel7
+      - foreman-nightly-rhel8
     releasers:
       - koji-foreman
     nightly_releaser: koji-foreman-jenkins
-    repoclosure_config: repoclosure/yum_el7.conf
+    repoclosure_config: repoclosure/yum.conf
     repoclosure_target_repos:
       rhel7:
         - el7-foreman-nightly
+      rhel8:
+        - foreman-nightly-el8
```

Comment:
this one should be el8-foreman-nightly, apologies for telling you the wrong thing.  This is based on what the repoid is in yum.conf `[el8-foreman-nightly]`